### ISSUE TITLE
Document Slack Setup

### ISSUE THREAD

**Issue by beverts312** [jfairley/github-slack-bot#148]
It would be helpful to know how to setup the bot from the slack app side

**Comment by Wagan8r** [comment]
> It would be helpful to know how to setup the bot from the slack app side

I agree. This would be very helpful. Somebody told me how to do it once, but then he left.

**Comment by jfairley** [comment]
I'll be interested in getting this set up at my new gig next week. That said, idk if they even have local servers or are all cloud. If the latter, I'll be diving into making it functional first... Because I no longer have access to an existing server runtime or slack account or github org with which I can test in production.
